File: TrabajoFinal/TrabajoFinal/Entity/Facturas.cs
namespace TrabajoFinal.Entity
{
    public class Facturas
    {
        public int Id { get; set; }
        public int? pedido_id { get; set; }
        public string? fecha { get; set; }
        public int total { get; set; }
        public bool pagado { get; set; }
    }
}

File: Examen2/src/models/Facturas.js
const mongoose = require('mongoose');
const facturasSchema = new mongoose.Schema({
    pedido: {type: String},
    total: { type: Number},
    fecha: { type: Date},
    pagado: { type: Boolean},
}, { collection: 'facturas' });

module.exports = mongoose.model('Facturas', facturasSchema);
